DOCS-3977 Updates the Marketplace Index Page + Adds Create a Marketplace Offering Doc Page (#15870)
DataDog/documentation: content/en/developers/marketplace/_index.md

type: documentation
further_reading:
- link: "https://www.datadoghq.com/partner/"
  tag: "Partner Network"
  text: "Datadog Partner Network"
- link: "https://www.datadoghq.com/blog/datadog-marketplace/"
  tag: "Blog"
  text: "Expand your monitoring reach with the Datadog Marketplace"
- link: "/developers/integrations/"
  tag: "Documentation"
  text: "Learn about Datadog integrations"
- link: "/developers/datadog_apps/"
  tag: "Documentation"
  text: "Learn about Datadog Apps"
---

## Overview

The Datadog Marketplace is a digital marketplace where Datadog Technology Partners can list their offerings to Datadog users. Datadog customers can access integration tiles on the [**Integrations** page][1] or the [**Marketplace** page][2]. 

{{< img src="developers/marketplace/marketplace_overview.png" alt="The Datadog Marketplace page" style="width:100%" >}}

> While the **Integrations** page includes data integrations and Datadog Apps built by Datadog and Technology Partners at no cost, the **Marketplace** page is a commercial platform for Datadog customers and Technology Partners to buy and sell a variety of offerings including data integrations, Datadog Apps, software, and professional services. 

Author: ```suggestion
While the **Integrations** page includes integrations and Datadog Apps built by Datadog and Technology Partners at no cost, the **Marketplace** page is a commercial platform for Datadog customers and Technology Partners to buy and sell a variety of offerings including Agent-based or API-based integrations, Datadog Apps, software, and professional services. 
```

## Join the Datadog partner network

> Before requesting access to the Datadog Marketplace, first apply to join the [Datadog Partner Network's][3] **Technology Partners** track. As a Datadog Technology Partner, you can either develop a [data integration][4] or a [Datadog App][5], or list your SaaS license or professional services offering. 

Author: ```suggestion
Before requesting access to the Datadog Marketplace, first apply to join the [Datadog Partner Network's][3] **Technology Partners** track. As a Datadog Technology Partner, you can either develop an [integration][4] or a [Datadog App][5], or list your SaaS license or professional services offering. 
```

2. On your personal homepage, click on the **Learn More** button under **Sandbox Access**.
3. Select **Request Sandbox Upgrade**.

<div class="alert alert-info">If you are already a member of a Datadog organization (including a trial org), you may need to switch to your newly created sandbox. For more information, see the <a href="https://docs.datadoghq.com/account_management/org_switching/">Account Management documentation</a>.</div>

Creating a developer sandbox may take up to one or two business days. Once your sandbox is created, you can [invite new members from your organization][7] to collaborate with.

## Explore learning resources

Once you've joined the Technology Partners track and requested a sandbox account, you can start learning about developing Datadog integrations and additional offerings by:

* Completing the on-demand [**Introduction to Datadog Integrations**][8] course on the [Datadog Learning Center][9].
* Joining the Marketplace Engineering Office Hours through the public [Datadog Slack channel][10].
* Reading the documentation about [Datadog Apps][5] if you want to build a custom widget that integrates external data and actions with Datadog dashboards. 

> * Reading the documentation about setting up an [OAuth 2.0 client][11] for data integrations and Datadog Apps.

Author: ```suggestion
* Reading the documentation about setting up an [OAuth 2.0 client][11] for Agent-based integrations and Datadog Apps.
```

## List an offering on Marketplace

All Technology Partners can list a free integration on the **Integrations** page or a commercial offering on the **Marketplace** page. Additional offerings on Datadog Marketplace may include:

Integrations
: Integrations that submit or pull third-party data through the Datadog Agent or an API.

Software licenses

> : Integrations that are informational and are tile-only listings on the Datadog Marketplace. 

Reviewer: ```suggestion
: SaaS licenses let you deliver and license software solutions to customers via the Datadog Marketplace. 
```

UI extensions or applications
: An application (such as a [Datadog App][5]) that are tile-only listings on the Datadog Marketplace.

Professional services

> : Integrations that are informational and are tile-only listings on the Datadog Marketplace. 

Reviewer: ```suggestion
: Professional Services let you offer your team's services for implementation, support, or management for a set period of time. 
```

> ### Review the example pull request

Author: ```suggestion
### Integration Extras and Marketplace
```

> While the process for building a data integration is identical in the [`integrations-extras`][12] and [`marketplace`][13] repositories, a Marketplace offering requires additional files and information such as pricing plans. 

Author: ```suggestion
While the process for building an integration is identical in the [`integrations-extras`][12] and [`marketplace`][13] repositories, a Marketplace offering requires additional files and information such as pricing plans. 
```

> While the process for building an integration is identical in the [`integrations-extras`][12] and [`marketplace`][13] repositories, a Marketplace offering requires additional files and information such as pricing plans.

Reviewer: ```suggestion
While the process for building an integration is identical for the Integrations page through [`integrations-extras`][12] and Marketplace page through the [`marketplace`][13] repositories, a Marketplace offering requires additional files and information such as pricing plans.
```

Just want to clarify which repo belongs to which page

### Request access to Marketplace

To request access to the private Marketplace repository, email <a href="mailto:marketplace@datadoghq.com">marketplace@datadoghq.com</a>.

Once you have been granted access, you can review an [example pull request][14] in the Marketplace repository with annotations and best practices.

## Further Reading

> {{< partial name="whats-next/whats-next.html" >}}

Reviewer: @alai97 Can we include a link to child page 'Develop a Marketplace offering' in this section?
